.circleci/config.yml

```diff
@@ -5,9 +5,9 @@ references:
     docker:
       - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
         environment:
-          GITHUB_TEAM_NAME_SLUG: claim-for-crown-court-defence
+          GITHUB_TEAM_NAME_SLUG: laa-get-paid
           APPLICATION_DEPLOY_NAME: laa-fee-calculator
-          DJANGO_SECRET_KEY: laa-fee-calculator
+          DJANGO_SECRET_KEY: real-secret-not-needed-here
```

Comment:
Can this be removed?

Reply:
@ddippolito perhaps, but if it is anything like the rails SECRET_KEY_BASE key (random key for signing cookies), which appears to be so, then it might be needed at build time by some DJANGO process. With rails it needs to exist for asset compilation even though it is not used.

will look into a bit further

Reply:
on the boxes the DJANGO_SECRET_KEY is set to the value of the secret. so build arg is overridden (which is good/as-i would-expect).

I'd be tempted to keep the line above as even if not needed now if a django process running at build time requires its presence now or in the future django updates it will be there. we do this in CCCD too for rails